playgrounds/vue/src/assets/base.css

```diff
@@ -0,0 +1,74 @@
+/* color palette from <https://github.com/vuejs/theme> */
```

Comment:
Minor suggestion.

This CSS was auto-generated by default.
We don't need it and it may confuse someone in the future.
Let's delete it.

Reply:
done